docs/wiki/feature/nightvision.md

```diff
@@ -1,24 +1,27 @@
 ---
 layout: wiki
 title: Nightvision
-description: Different types of NVGs with different levels of quality.
+description: Adds realistic effects to NVGs.
 group: feature
 category: equipment
 parent: wiki
 mod: ace
 version:
   major: 3
-  minor: 0
+  minor: 12
```

Comment:
Hmm, yes, but I'd also add a note saying it was changed in 3.12.0, not added.

Reply:
Elaborate ?

Reply:
@Tittoffer Night Vision component has been present since 3.0.0, but only changed in 3.12.0, that's what I meant. Sorry for replying late, didn't see that!